lib/screens/v2/widget_instance/elevator_status.ex

```diff
@@ -260,6 +291,7 @@ defmodule Screens.V2.WidgetInstance.ElevatorStatus do
     end)
   end
 
+  # Seems familiar... keep an eye out for duplicate,
```

Comment:
I think I cut some corners here because elevator_closure alerts have a simpler set of possible informed entity shapes. I believe it's always `{facility: "facility id", stop: "platform stop id or parent station id"}`.

A general function would need to handle more cases.